src/printers/relationships.js

```diff
@@ -33,37 +34,46 @@ export const exporter = (node: RawNode): string => {
   return str;
 };
 
-export const imports = (node: ImportNode, moduleName: string): string => {
-  let str = "import type ";
+// TODO: move import here
```

Comment:
This is commented because import node is printing itself in class, but should  be moved to printers folder